tsl/src/remote/dist_ddl.c

```diff
@@ -390,9 +420,23 @@ dist_ddl_preprocess(ProcessUtilityArgs *args)
 			break;
 
 		case T_CreateTrigStmt:
+		{
+			CreateTrigStmt *stmt = castNode(CreateTrigStmt, args->parsetree);
+
+			if (stmt->funcname != NIL)
+			{
+				Oid funcargtypes[1];
+				Oid funcid = LookupFuncName(stmt->funcname,
+											0 /* nargs */,
+											funcargtypes /* passing NULL is not allowed */,
+											false /* missing ok */);
+				Datum datum = DirectFunctionCall1(pg_get_functiondef, ObjectIdGetDatum(funcid));
```

Comment:
LookupFuncName could error and not return the correct oid. That case needs handling?
```
/*
 * LookupFuncNameInternal
 *      Workhorse for LookupFuncName/LookupFuncWithArgs
 *
 * In an error situation, e.g. can't find the function, then we return
 * InvalidOid and set *lookupError to indicate what went wrong.
 *
 * Possible errors:
 *  FUNCLOOKUP_NOSUCHFUNC: we can't find a function of this name.
 *  FUNCLOOKUP_AMBIGUOUS: nargs == -1 and more than one function matches.
 */
````

Reply:
We used missing_ok=false, so the OID should be valid.

Reply:
Added assert anyway.